src/chocolatey/infrastructure.app/configuration/ConfigurationOptions.cs

```diff
@@ -58,28 +56,25 @@ private static IConsole Console
                                                                     Action helpMessage)
         {
             IList<string> unparsedArguments = new List<string>();
+            var optionSet = new OptionSet 
+            {{
+                "?|help|h", "Prints out the help menu.", 
+                option => configuration.HelpRequested = option != null
+            }};
 
-            // add help only once
-            if (_optionSet.Count == 0)
-            {
-                _optionSet
-                    .Add("?|help|h",
-                         "Prints out the help menu.",
-                         option => configuration.HelpRequested = option != null);
-            }

```

Comment:
Yeah, this looks bad. You don't come through there twice b/c ConfigurationBuilder is your only call through, but a Console application comes through there twice. Please revert the changes to this file https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/runners/ConsoleApplication.cs#L51